storage/badger/operation/common.go

```diff
@@ -386,3 +387,11 @@ func Fail(err error) func(*badger.Txn) error {
 		return err
 	}
 }
+
+// terminateOnFullDisk panics to crash the node if an insert / batchInsert / update fail due to a full disk
+func terminateOnFullDisk(err error) error {
+	if err != nil && errors.Is(err, syscall.ENOSPC) {
+		panic("disk full, terminating node...")
```

Comment:
```suggestion
		panic(fmt.Sprintf("disk full, terminating node: %s", err.Error())
```
include full error text